src/util/orders.test.ts

```diff
@@ -77,3 +76,54 @@ describe('buildOrder', () => {
         );
     });
 });
+describe('orderDetails', () => {
+    it('orderDetailsFeeEther should return a BigNumber ', () => {
```

Comment:
Do we need this kind of tests if we have types?

Reply:
I deleted this and added a better test that checks a result against a mocked dolar vallue